### PR TITLE
Add eiffel-goer project

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

#99 

### Description of the Change

Add Eiffel-Goer to PROJECTS and CONTACT

### Alternate Designs

N/A

### Benefits

Communications is key

### Possible Drawbacks

N/A

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias Persson <tobias.persson@axis.com>

### CODE REVIEW

**Review by magnusbaeck — COMMENTED** [review]
Should we try to keep these tables in alphabetical order? Also, should we stick the convention in CONTACT.md of having the bare repo name in the first column? I know that we've already broken the mold with ETOS, but that's on the other hand an aggregation of multiple repos.

Unrelated to your PR I'm struggling to understand why we have nearly identical tables in both CONTACT.md and PROJECTS.md.

**Comment by t-persson** [comment]
> Should we try to keep these tables in alphabetical order? Also, should we stick the convention in CONTACT.md of having the bare repo name in the first column? I know that we've already broken the mold with ETOS, but that's on the other hand an aggregation of multiple repos.
> 
> Unrelated to your PR I'm struggling to understand why we have nearly identical tables in both CONTACT.md and PROJECTS.md.

Updated the name of the project in CONTACT.md. The rest should probably be discussed in an issue instead :)

**Comment by m-linner-ericsson** [comment]
@magnusbaeck I created https://github.com/eiffel-community/community/issues/106, feel free to change as much as you like 😃

File: PROJECTS.md [hunk 1]
```diff
@@ -50,3 +50,4 @@ Please see [project lifecycle process](./PROJECT_LIFECYCLE.md) for the project l
 | [Eiffel Sepia](https://github.com/eiffel-community/eiffel-sepia) | | [eiffel-sepia-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-sepia-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
 | [Eiffel Store](https://github.com/eiffel-community/eiffel-store) | A persistence solution for Eiffel events which accepts live view of Eiffel events. | [eiffel-store-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-store-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
 | [Eiffel Vici](https://github.com/eiffel-community/eiffel-vici) | Eiffel Vici dynamically visualizes Eiffel events retrieved from an Eiffel Event Persistence Service. | [eiffel-vici-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-vici-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
+| [Eiffel Goer](https://github.com/eiffel-community/eiffel-goer) | Eiffel Goer is an open source implementation of the event repository API. | [eiffel-goer-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-goer-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
```
Comment (m-linner-ericsson) [inline PROJECTS.md:53]: Would explaining the name help? Could update it to something like
```suggestion
| [Eiffel Goer](https://github.com/eiffel-community/eiffel-goer) | Eiffel Goer is an implementation of the event repository API in GO. | [eiffel-goer-maintainers](https://github.com/orgs/eiffel-community/teams/eiffel-goer-maintainers/members) | [Sandbox](./PROJECT_LIFECYCLE.md#stage-sandbox) |
```

If you get my idea?

Comment (t-persson) [reply PROJECTS.md]: Done

**Review by m-linner-ericsson — APPROVED** [review]

**Review by e-backmark-ericsson — APPROVED** [review]
The project names should be in alphabetical order, but I guess we sort that out later.

**Pull request merged**